### DISCUSSION TITLE
bllip-parser Error downloading model (101 Network is unreachable)

### DISCUSSION THREAD

**Discussion by liutaonuist** [community/community#38275] [Other Feature Feedback, Questions, & Ideas] [answered]
When i run wget
 https://www.dropbox.com/s/ulcfs7my1ifriuu/BLLIP-WSJ-PTB3.tar.bz2?dl=1:

 Fetching model: WSJ-PTB3 from
 https://www.dropbox.com/s/ulcfs7my1ifriuu/BLLIP-WSJ-PTB3.tar.bz2?dl=1
 Traceback (most recent call last):
 File "", line 1, in
 File
 "/home/lzm/.local/lib/python2.7/site-packages/bllipparser/RerankingParser.py",
 line 897, in fetch_and_load
 verbose)
 File
 "/home/lzm/.local/lib/python2.7/site-packages/bllipparser/ModelFetcher.py",
 line 124, in download_and_install_model
 reporthook=status_func)
 File "/usr/lib/python2.7/urllib.py", line 245, in retrieve
 fp = self.open(url, data)
 File "/usr/lib/python2.7/urllib.py", line 213, in open
 return getattr(self, name)(url)
 File "/usr/lib/python2.7/urllib.py", line 443, in open_https
 h.endheaders(data)
 File "/usr/lib/python2.7/httplib.py", line 1053, in endheaders
 self._send_output(message_body)
 File "/usr/lib/python2.7/httplib.py", line 897, in _send_output
 self.send(msg)
 File "/usr/lib/python2.7/httplib.py", line 859, in send
 self.connect()
 File "/usr/lib/python2.7/httplib.py", line 1270, in connect
 HTTPConnection.connect(self)
 File "/usr/lib/python2.7/httplib.py", line 836, in connect
 self.timeout, self.source_address)
 File "/usr/lib/python2.7/socket.py", line 575, in create_connection
 raise err
 IOError: [Errno socket error] [Errno 101] Network is unreachable

could you tell me how you solve this problem？ thank you very much.

**Comment by david-engelmann** [comment, marked as answer]
It's probably best to ask this question in a more relevant area such as the [bllip-parser repository](https://github.com/BLLIP/bllip-parser).

**Comment by liutaonuist** [comment]
&nbsp;Though the model exist at the link,the web link cannot be opened and the network is congested during the download，so whether there are some&nbsp;measures sloving this problem？


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "community/community"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月4日(星期五) 晚上10:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [community/community] bllip-parser Error downloading model (101 Network is unreachable) (Discussion #38275)





 
It's probably best to ask this question in a more relevant area such as the bllip-parser repository.

**Comment by airtower-luna** [comment]
"Network is unreachable" means that the machine where you run the command doesn't have a network connection to the server you want to reach. Most likely that's a network configuration problem.

However the error message also shows that you're still using Python 2, which has been [out of support for almost 3 years](https://www.python.org/doc/sunset-python-2/) now, so all bets are off. :sweat_smile:

**Comment by liutaonuist** [comment]
Traceback (most recent call last):
&nbsp; File "amr_parsing.py", line 435, in <module&gt;
&nbsp; &nbsp; main()
&nbsp; File "amr_parsing.py", line 168, in main
&nbsp; &nbsp; instances = preprocess(amr_file,START_SNLP=True,INPUT_AMR=args.amrfmt, PRP_FORMAT=args.prpfmt)
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/preprocessing.py", line 504, in preprocess
&nbsp; &nbsp; dparser.parse(tok_sent_filename)
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/depparser.py", line 31, in parse
&nbsp; &nbsp; rrp = RerankingParser.from_unified_model_dir(path_to_model)
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/bllipparser/RerankingParser.py", line 879, in from_unified_model_dir
&nbsp; &nbsp; rrp.load_parser_model(parser_model_dir, **parsing_options)
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/bllipparser/RerankingParser.py", line 592, in load_parser_model
&nbsp; &nbsp; parser.loadModel(model_dir)
AttributeError: type object 'CharniakParser' has no attribute 'loadModel'



note that how to solve this problem?&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "community/community"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月5日(星期六) 下午5:32
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [community/community] bllip-parser Error downloading model (101 Network is unreachable) (Discussion #38275)





 
"Network is unreachable" means that the machine where you run the command doesn't have a network connection to the server you want to reach. Most likely that's a network configuration problem.
 
However the error message also shows that you're still using Python 2, which has been out of support for almost 3 years now, so all bets are off. 😅

**Comment by liutaonuist** [comment]
Traceback (most recent call last):
&nbsp; File "amr_parsing.py", line 435, in <module&gt;
&nbsp; &nbsp; main()
&nbsp; File "amr_parsing.py", line 168, in main
&nbsp; &nbsp; instances = preprocess(amr_file,START_SNLP=True,INPUT_AMR=args.amrfmt, PRP_FORMAT=args.prpfmt)
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/preprocessing.py", line 504, in preprocess
&nbsp; &nbsp; dparser.parse(tok_sent_filename)
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/depparser.py", line 24, in parse
&nbsp; &nbsp; from bllipparser.ModelFetcher import download_and_install_model
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/bllipparser/__init__.py", line 400, in <module&gt;
&nbsp; &nbsp; from .RerankingParser import RerankingParser, Tree, Sentence, tokenize
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/bllipparser/RerankingParser.py", line 19, in <module&gt;
&nbsp; &nbsp; from . import CharniakParser as parser
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/bllipparser/CharniakParser.py", line 17, in <module&gt;
&nbsp; &nbsp; _CharniakParser = swig_import_helper()
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/bllipparser/CharniakParser.py", line 16, in swig_import_helper
&nbsp; &nbsp; return importlib.import_module('_CharniakParser')
&nbsp; File "/root/autodl-tmp/CLEVE-main/camr/importlib/__init__.py", line 37, in import_module
&nbsp; &nbsp; __import__(name)
ImportError: No module named _CharniakParser



could you tell me how to solve this problem？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "community/community"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月5日(星期六) 下午5:32
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [community/community] bllip-parser Error downloading model (101 Network is unreachable) (Discussion #38275)





 
"Network is unreachable" means that the machine where you run the command doesn't have a network connection to the server you want to reach. Most likely that's a network configuration problem.
 
However the error message also shows that you're still using Python 2, which has been out of support for almost 3 years now, so all bets are off. 😅

**Comment by queenofcorgis** [comment]
Hi @Liutaonuist,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your additional post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
